src/components/panel/editForm/Panel.edit.display.js

```diff
@@ -135,6 +135,16 @@ export default [
           next: true
         },
       },
+      {
+        weight: 55,
+        label: 'Navigate Pages & Save on Enter',
```

Comment:
We already have an option to "Press Enter to Submit"... We should just make this option "Navigate Wizard on Enter". If they wish to navigate and submit on Enter, then they should have both of those options checked.